doc/manuals/orion-api.md

```diff
@@ -674,6 +674,8 @@ the subscriptions based in alteration type features (see [Subscription based in
    * `entityChange` if the update that triggers the notification was an update with an actual change or not an actual change but with `forcedUpdate` in use
    * `entityDelete` if the update that triggers the notification was a entity delete operation
 
+* `servicePath` (type: `Text`): specifies the service path associated to a given entity (See subsection [Service Path](#service-path) for more information)
+
 Like regular attributes, they can be used in `q` filters and in `orderBy` (except `alterationType`).
```

Comment:
@chetan-NEC how `servicePath` behaves with `q` and `orderBy`? Could you add a couple of thest (one for `q` and other for `orderBy`) in PR https://github.com/telefonicaid/fiware-orion/pull/4329, please?

Depending on the result, we would add `servicePath` to the "except" list or not.

Reply:
@fgalan My understanding is that I must include test cases in PR #4329 for the servicePath with q and orderBy. Please confirm my understanding.

Reply:
I confirm your understanding. Thanks!

Reply:
@fgalan In PR #4329, I've added test cases as you suggested. Please review the PR and let me know if you have any suggestions.